src/win/process.c

```diff
@@ -169,7 +169,10 @@ static WCHAR* search_path_join_test(const WCHAR* dir,
                                     size_t cwd_len) {
   WCHAR *result, *result_pos;
   DWORD attrs;
-  if (dir_len > 2 && dir[0] == L'\\' && dir[1] == L'\\') {
+  if (dir_len > 2 && (
+	(dir[0] == L'\\' && dir[1] == L'\\') ||
+	(dir[0] == L'/' && dir[1] == L'/') 
```

Comment:
```suggestion
	(dir[0] == L'\\' && dir[0] == L'/') ||
	(dir[1] == L'\\' && dir[1] == L'/') 
```

Reply:
@vtjnash I cannot approve your suggestion.
`(dir[0] == L'\\' && dir[0] == L'/')` will never be true... idem for `dir[1]`.

Reply:
```suggestion
	(dir[0] == L'\\' || dir[0] == L'/') &&
	(dir[1] == L'\\' || dir[1] == L'/') 
```

Reply:
bump

Reply:
@vtjnash your second suggestion will allow paths starting with "\/" and "/\".
I think that windows only support "//" and "\\".

Reply:
Those paths worked when I tested windows

Reply:
If you are able to test and confirm which is correct, I can probably get this merged before the current release